### ISSUE TITLE
MatSlideToggle duplicate id

### ISSUE THREAD

**Issue by Lucidize** [SamProf/MatBlazor#845]
**Describe the bug**
The MatSlideToggle component contains a duplicate id which is breaking accessibility.

[F77: Failure of Success Criterion 4.1.1 due to duplicate values of type ID
](https://www.w3.org/TR/WCAG20-TECHS/F77.html)
Duplicate values of type ID can be problematic for user agents that rely on this attribute to accurately convey relationships between different parts of content to users

**To Reproduce**
```
<MatSlideToggle @bind-Value="@Val1"></MatSlideToggle>
        <span>@Val1</span>

@code
    {
        private bool Val1 = true;
    }
```

**Blazorfiddle link**
https://blazorfiddle.com/s/vcp381ki

**Expected behavior**
The id should be unique but in the MatSlideToggle component they are reused.

**Screenshots**
https://imgur.com/3wtCjSZ
https://imgur.com/e1pWWBa

**Comment by SamProf** [comment]
fixed by you
